webext-options-sync.js

```diff
@@ -1,8 +1,15 @@
 // https://github.com/bfred-it/webext-options-sync
 
 class OptionsSync {
-	constructor(storageName = 'options') {
-		this.storageName = storageName;
+	constructor(options = {}) {
```

Comment:
`storageName` used to be a string, this needs to detect if a user passed a string and change it to an object

Reply:
Something like this will do?

```js
constructor(options = {}) {
	if (typeof options === 'string') {
		options = {
			storageName: options
		}
	}

	this.storageName = options.storageName || 'options';
	if (options.logging === false) {
		this._log = () => {};
	}
}
```

Reply:
Yes, that'd work

Reply:
Fixed! :)